includes/class-sensei-utils.php

```diff
@@ -2714,6 +2714,34 @@ public static function is_teacher_archive_page() {
 		return false;
 	}
 
+	/**
+	 * Gets the contents of the icon file at assets/images/<name>.svg
+	 * for the given name. Or empty string if file not found.
+	 *
+	 * @param string $name The name of the icon file at "assets/images/<name>.svg".
+	 * @return string The icon markup.
+	 */
+	public static function icon( string $name = '' ) {
+		$dir     = realpath( __DIR__ . '/../assets/images' );
+		$file    = "{$dir}/{$name}.svg";
+		$content = '';
+
+		// Read file inside try/catch in case the
+		// icon file is not there.
+		try {
+			ob_start();
+			include $file;
+			$content = ob_get_clean();
```

Comment:
Would `file_get_contents` work better here?

Reply:
I actually used `get_file_contents` first. But then there were a linting warning that told me to not use it. But I guess it was not related to this case.
<img width="1281" alt="Screen Shot 2021-11-22 at 3 26 34 PM" src="https://user-images.githubusercontent.com/2578542/142856316-adea55e4-e452-493c-9b8b-33f0e5f1bc76.png">

Updated it now to use `file_get_contents`. 👍

Reply:
OK, now the lint checks are failing https://github.com/Automattic/sensei/runs/4285969450?check_suite_focus=true#step:8:29

Should I disable that rule in that particular place?

Reply:
Yup it should be fine to disable it for a local file, like here:

https://github.com/Automattic/sensei/blob/721bc25c11b1bba7e1ed1b54f1c5b554d04ba2a8/includes/class-sensei-updates.php#L354-L355

Reply:
Thanks! Done. 👍